### ISSUE TITLE
(v9) Badge: badge does not show the full text when zoom 200

### ISSUE THREAD

**Issue by tringakrasniqi** [microsoft/fluentui#21130]
### Environment Information

- **Package version(s)**: v9 beta
- **Browser and OS versions**: Edge

### Describe the issue:
Adding a long text to badge and zooming to 200, the text is not readable:

![zoom-200-long text badge](https://user-images.githubusercontent.com/12565826/147583817-467c43f4-31e6-4fdf-a74c-f290080dc6d7.png)
the badge does not resize to fit the text.

### Please provide a reproduction of the issue in a codepen:
Codesandbox: https://codesandbox.io/s/badge-a11y-review-bhu70
(open in a new window with only the components)

### Expected behavior:
The design spec does not give any specifics on this, however the badge could maybe be resized to fit the text. To be discussed (might be a non-issue).

**Comment by tringakrasniqi** [comment]
This could be addressed by updating documentation to state that Badges do not support long text and it should not be used for long text.
